core/templates/dev/head/pages/exploration_player/tutor_card_directive.html

```diff
@@ -0,0 +1,267 @@
+      display: none;
+    }
+
+    .conversation-skin-tutor-card .instructions-button {
+        background: inherit;
+        border: none;
+    }
+
+    /*TODO: do we need?*/
+    .conversation-skin-tutor-card.animate-card-width {
+      -webkit-transition: width 500ms;
+      transition: width 500ms;
+    }
+
+    .conversation-skin-future-tutor-card {
+      left: -30000px;
+      position: fixed;
+      top: -30000px;
+    }
+
+    .conversation-skin-tutor-card-top-section {
+      position: relative;
+    }
+
+    .conversation-skin-tutor-card-top-section {
+      padding: 0 20px;
+    }
+
+    .conversation-skin-tutor-card-top-content > p {
+      line-height: 28px;
+      margin-bottom: 18px;
+      margin-top: 18px;
+    }
+
+
+    .conversation-skin-tutor-card-top-content {
+      width: 100%;
+    }
+
+    .conversation-skin-tutor-card,
+    .conversation-skin-future-tutor-card {
+      background: #fff;
+      border-radius: 2px;
+      box-shadow: 0 1px 2px rgba(0, 0, 0, 0.24), 0 1px 3px rgba(0, 0, 0, 0.12);
+      padding-top: 20px;
+      text-align: left;
+    }
+
+    .conversation-skin-tutor-card-middle-section .conversation-skin-learner-answer-content {
+      background-color: rgba(236,239,241,0.4);
+    }
+
+    .conversation-skin-tutor-card-middle-section .conversation-skin-oppia-feedback-content {
+      background-color: rgba(224,242,241,0.4);
+    }
+
+    .conversation-skin-tutor-card-middle-section .conversation-skin-oppia-feedback-content::before,
+    .conversation-skin-tutor-card-middle-section .conversation-skin-learner-answer-content::before {
+      display: none;
+    }
+
+    .conversation-skin-tutor-card.ng-hide {
+      opacity: 0;
+      transform: translateX(-1000px);
+    }
+
+    .conversation-skin-tutor-card-content {
+      word-wrap: break-word;
+    }
+
+    .conversation-skin-responses-dropdown-container {
+      border-bottom: 1px solid #ccc;
+      cursor: pointer;
+      line-height: 0.1em;
+      margin: 8px 20px 20px 20px;
+      text-align: center;
+    }
+
+    .conversation-skin-responses-dropdown-text {
+      background-color: #fff;
+      color: #888;
+      font-size: 0.8em;
+      padding: 0 10px;
+    }
+
+    .conversation-skin-responses-dropdown-icon {
+      -moz-transition: all 0.2s;
+      -o-transition: all 0.2s;
+      -webkit-transition: all 0.2s;
+      font-size: 10px;
+      margin-bottom: -4px;
+      transition: 0.2s;
+    }
+
+    .conversation-skin-responses-dropdown-icon-rotated {
+      -moz-transform: rotate(90deg);
+      -o-transform: rotate(90deg);
+      -webkit-transform: rotate(90deg);
+      transform: rotate(90deg);
+    }
+
+    .conversation-skin-inline-interaction {
+      background-color: rgba(236,239,241,1);
+      border-bottom-left-radius: 2px;
+      border-bottom-right-radius: 2px;
+      margin-top: 8px;
+      padding: 16px 20px;
+      position: relative;
+    }
+
+    /* TODO: do we need? */
```

Comment:
See if this makes any difference to how the avatars appear for things like numeric input and text input when the user answers a question.